### PR TITLE
Fixed errors and warnings

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/10583

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -40,7 +40,12 @@ apt-get install -y  \
   rsync=3.1.0* \
   psmisc=22.20* \
   vim=2:7.4.052*
+  
+#update-rc.d: warning: default stop runlevel arguments (0 1 6) do not match rsync Default-Stop values (none)
```
Comment (ric03uec) [inline install.sh]: we dont need to paste the warning here. just remove all the warnings and comments for now

Comment (vijayreddy1991) [reply install.sh]: ok

File: install.sh [hunk 2]
```diff
@@ -40,7 +40,10 @@ apt-get install -y  \
   rsync=3.1.0* \
   psmisc=22.20* \
   vim=2:7.4.052*
+  
+# They not resolved till now - https://github.com/Microsoft/WSL/issues/2782
```
Comment (ric03uec) [inline install.sh]: - `rsync throws a warning that is not resolved yet: <link>`

File: install.sh [hunk 3]
```diff
@@ -40,7 +40,10 @@ apt-get install -y  \
...
 
+# This types of warnings is negligible.If we upgrade python it wont show this type of warnings
```
Comment (ric03uec) [inline install.sh]: `python throws a few warnings that can be ignored. New versions of python do not throw these warnings`

File: install.sh [hunk 4]
```diff
@@ -56,6 +59,8 @@ add-apt-repository ppa:git-core/ppa -y
 apt-get update
 apt-get install  -y git=1:2.17.0*
 
+
+# This error is negligible - https://github.com/git-lfs/git-lfs/issues/2837
```
Comment (ric03uec) [inline install.sh]: `This throws a warning that can be ignored: <link>`

**Pull request merged**